Sources/MapboxMaps/Location/LocationOptions.swift

```diff
@@ -24,6 +24,10 @@ public struct LocationOptions: Equatable {
     /// Sets the type of puck that should be used
     public var puckType: PuckType?
 
+    /// Specifies if `Pucks` should use `Heading` or `Course` for the bearing
+    /// This is an experimental option
+    public var bearingType: BearingType = .heading
```

Comment:
could this be renamed to something a bit different? `puckOrientationSource` ?

Reply:
I feel like bearingType is more descriptive

Reply:
It feels quite vague to be honest.

Reply:
Are you okay with `puckBearingSource`